ConTabs/Table.cs

```diff
@@ -7,11 +7,30 @@
 
 namespace ConTabs
 {
-    [DebuggerDisplay("Table with {Columns.Count} available columns")]
-    public partial class Table<T> where T:class
-    {
-        public Columns Columns { get; set; }
-        internal List<Column> _colsShown => Columns.Where(c => !c.Hide).ToList();
+	[DebuggerDisplay("Table with {Columns.Count} available columns")]
+	public partial class Table<T> where T : class
+	{
+		public Columns Columns { get; set; }
+		public Alignment HeaderAlignment { get; set; }
```

Comment:
I was a little surprised to see header alignment controlled at the table level. I think intuitive I was expecting header alignment to follow the columns. E.g. if I set a column to right-aligned, the header for that column would also be right aligned.

Reply:
Thank you for your feedback, really appreciate it!

My reasoning behind this was that you would usually want all the headers to have the same alignment, couldn't really think of a scenario where you would want to change the alignment of an individual header. 

For example if you set the column alignment to right because of numbers with decimal places like this:
```
+--------+--------+
| Apple  | Orange |
+--------+--------+
|  12.50 | ABC    |
| 123.21 | AB     |
+--------+--------+
```

It would be weird if the header would be aligned to the right as well. But then again that's just what I thought :) Another option is to not align the headers at all and only use the alignment setting for the rows. 

Let me know what you think, I'd be happy to change it!

Reply:
Y'know what? I think you may actually be right. My colleague agrees with you too. Yep, OK, I'm convinced.